k8s/utils/symphony-api.go

```diff
@@ -63,8 +63,11 @@ func K8SComponentSpecToAPIComponentSpec(component k8smodel.ComponentSpec) (apimo
 	if err != nil {
 		return apimodel.ComponentSpec{}, err
 	}
-	componentSpec.Properties = make(map[string]interface{})
```

Comment:
Sidecar also has a optional properties, please also fix that.

Reply:
fixed